runtests.py

```diff
@@ -127,7 +127,7 @@ def main(argv):
     parser.add_argument("args", metavar="ARGS", default=[], nargs=REMAINDER,
                         help="Arguments to pass to Nose, Python or shell")
     args = parser.parse_args(argv)
-
+    print(args)
```

Comment:
@seberg, you not gonna keep it, right?

Reply:
Ops, no, of course not, thanks :). I was confused about how argparse does things... and I think now it does it "the same way" with `--bench` (at least kind of).

Reply:
I do the same most of the time, git hook for detecting such case like this may help u.

Reply:
Maybe (some general hooks on NumPy that are easy to enable could be neat too)... anyway thanks @seiko2plus, I will put this in, its easy enough to change if its not yet ideal...